src/NServiceBus.AcceptanceTests/Routing/NativePublishSubscribe/When_multi_subscribing_to_a_polymorphic_event.cs

```diff
@@ -59,7 +59,11 @@ public class Subscriber : EndpointConfigurationBuilder
         {
             public Subscriber()
             {
-                EndpointSetup<DefaultServer>();
+                EndpointSetup<DefaultServer>(publisherMetadata: metadata =>
+                        {
+                            metadata.RegisterPublisherFor<IMyEvent>(typeof(Publisher1));
+                            metadata.RegisterPublisherFor<MyEvent2>(typeof(Publisher2));
+                        });
```

Comment:
since this tests specifies ` .Repeat(r => r.For<AllTransportsWithCentralizedPubSubSupport>())` there should be no need to register publishers?